src/Controls/tests/Xaml.UnitTests/OnPlatformOptimization.xaml

```diff
@@ -3,13 +3,12 @@
 			 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
 			 x:Class="Microsoft.Maui.Controls.Xaml.UnitTests.OnPlatformOptimization">
 	<StackLayout>
-		<Border>
+		<Grid>
 			<Label x:Name="label0"
 				Text="{OnPlatform Android='john', iOS='paul', Default='ringo' }"
 				Padding="{OnPlatform iOS='2,3,4,5'}" />
-		    <Border.StrokeShape>
-			<RoundRectangle CornerRadius="12"/>
-		    </Border.StrokeShape>
```

Comment:
Why removing the border? Maybe it was added for a test?

Reply:
It looked like it was a copy-paste, from another test -- it wasn't used.

I changed the `Border` to a `Grid` so we could have two labels.